pkg/harbor/project.go

```diff
@@ -315,6 +316,19 @@ func (p *project) GetReplicationRules(ctx context.Context, trigger globalregistr
 			results = append(results, replRule)
 		}
 	}
+
+	cronJobFactory, err := config.NewCjFactory(p.registry, p)
+	if err != nil {
+		return nil, err
+	}
+	cronJobReplicationRules, err := cronJobFactory.GetAllCronJobsForProject(ctx, p, p.registry.GetName())
+	if err != nil {
+		return nil, err
+	}
```

Comment:
With this in place we need to have Kubernetes configured. This makes the old CLI-style operation impossible.